src/Symfony/Component/Form/Test/TypeTestCase.php

```diff
@@ -34,6 +35,22 @@ protected function setUp()
         $this->builder = new FormBuilder(null, null, $this->dispatcher, $this->factory);
     }
 
+    protected function getExtensions()
```

Comment:
I'm still against this, as said in #21780 we should add the extension overhead on all base tests, so this should live in a specific type test case like `ValidationTypeTestCase`.